packages/mongodb-dialects/java-driver/src/test/kotlin/com/mongodb/jbplugin/dialects/javadriver/glossary/NamespaceExtractorTest.kt

```diff
@@ -438,4 +438,67 @@ public class JavaDriverRepository extends BaseRepository {
         assertEquals("myDatabase", namespace.database)
         assertEquals("myCollection", namespace.collection)
     }
+
+    @ParsingTest(
+        "Repository.java",
+        """
+import com.mongodb.client.FindIterable;
```

Comment:
This test looks not relevant to the changes here, or is it somehow?

Reply:
Ah no, it's not relevant, forgot to remove it after a PoC. Thanks for realising!